src/auto_schedule/auto_schedule.cc

```diff
@@ -329,44 +327,55 @@ std::vector<Ref<Sketch>> AutoSchedule::evolutionarySearch(size_t outSize) {
             break;
         }
 
-        while (v2.size() < EVOLUTIONARY_SEARCH_POPULATION) {
-            std::vector<Ref<Sketch>> now(EVOLUTIONARY_SEARCH_POPULATION);
-            if (verbose_ >= 1) {
-                logger() << "evo " << v2.size() << std::endl;
-            }
-            // Use static schedule for deterministic random numbers
+        auto prob =
+            getProbFromPredict(pred); // Sketches with higher performance are
+                                      // picked with higher probabilities
+        std::discrete_distribution<int> probDist(prob.begin(), prob.end());
+
+        v2.resize(EVOLUTIONARY_SEARCH_POPULATION);
```

Comment:
Maybe declare `v2` right here? It seems to have not been used prior to this `resize`.

Reply:
OK I see it reuses space through exchanging `v1` and `v2`.